timescaledb/how-to-guides/update-timescaledb/update-timescaledb.md

```diff
@@ -0,0 +1,68 @@
+# Updating major versions of TimescaleDB [](update)
+
+Use these instructions to update TimescaleDB within the same major release
+version (eg. TimescaleDB 2.1->2.2 or 1.7->1.7.4). If you need to upgrade between
+TimescaleDB 1.x and 2.x, please see our [separate upgrade document][update-tsdb-2] 
+for more details instructions.
+
+### TimescaleDB release compatibility
+
+TimescaleDB currently supports the following PostgreSQL releases. If you are not currently running
+a compatible release, please upgrade before updating TimescaleDB.
+
+ TimescaleDB Release |   Supported PostgreSQL Release
+ --------------------|-------------------------------
+ 1.7                 | 9.6, 10, 11, 12
+ 2.0                 | 11, 12
+ 2.1+                | 11, 12, 13
+
+If you need to upgrade PostgreSQL first, please see [our documentation][upgrade-pg].
+
+### Update TimescaleDB
+
+Software upgrades use PostgreSQL's `ALTER EXTENSION` support to update to the
+latest version. TimescaleDB supports having different extension
+versions on different databases within the same PostgreSQL instance. This
+allows you to update extensions independently on different databases. The
+upgrade process involves three-steps:
+
+1. We recommend that you perform a [backup][] of your database via `pg_dump`.
```

Comment:
```suggestion
1. Perform a [backup][] of your database via `pg_dump`.
```

Adding the "we recommend" weakens the statement here. We definitely want them to do this.